### PR TITLE
Refactor unique method to use exists without empty string

### PR DESCRIPTION
**Pull request by gravetii**

### CODE REVIEW

**Comment by lenguyenthanh** [comment]
this is much better, thanks!

File: modules/ingestor/src/main/scala/package.scala [hunk 1]
```diff
@@ -32,8 +32,7 @@ extension [A](changes: List[ChangeStreamDocument[A]])
   def unique: List[ChangeStreamDocument[A]] =
     changes
       .foldRight(List.empty[ChangeStreamDocument[A]] -> Set.empty) { case (change, p @ (acc, ids)) =>
-        val id = change.docId.getOrElse("")
-        if !ids.contains(id) && id != ""
+        if change.docId.exists(ids.contains)
```
Comment (gravetii) [inline modules/ingestor/src/main/scala/package.scala:35]: hi @lenguyenthanh this should actually be `if change.docId.exists(!ids.contains)`

Comment (gravetii) [reply modules/ingestor/src/main/scala/package.scala:35]: pushed the patch here - https://github.com/gravetii/lila-search/pull/2

Comment (gravetii) [reply modules/ingestor/src/main/scala/package.scala:35]: pushed the patch here - https://github.com/lichess-org/lila-search/pull/308

**Pull request merged**